readme.txt

```diff
@@ -87,6 +87,9 @@ A: It sends a single, polite email to commenters without Gravatars, inviting the
 
 == Changelog ==
 
+= 0.10.0 =
```

Comment:
This fixing will be release with [the comment enhancement feature](https://github.com/Automattic/gravatar-enhanced/pull/64)